Automattic/wp-calypso
# Stats: Replace comparing text with icon and tooltips for the 7-day highlights section

#### Proposed Changes

* Introduce the `info` icon to replace the comparing text for the `7-day highlights`.
* Add tooltips with the comparing text and charts for the `info` icon.

#### Testing Instructions



* Spin up this change with the Calypso Live link.
* Navigate to the `Stats` > `Traffic` page.
* Ensure the comparing text on the heading of `7-day highlights` is replaced with the `info` icon.
* Ensure the tooltips for the `info` icon display in line with the design.

|Before|After|
|-|-|
|<img width="660" alt="截圖 2023-02-06 上午10 49 49" src="https://user-images.githubusercontent.com/6869813/216872815-4fd19bb8-2765-4edb-bc1d-249eda3ec90e.png">|<img width="667" alt="截圖 2023-02-06 上午10 44 39" src="https://user-images.githubusercontent.com/6869813/216872888-d66aca3b-d736-471c-9f95-fee64cc8c5c8.png">|

#### Pre-merge Checklist





Related to #70089
